echopype/tests/visualize/test_plot.py

```diff
@@ -172,6 +172,6 @@ def test_plot_mvbs(
         if 'azfp_cal_type' in range_kwargs:
             range_kwargs.pop('azfp_cal_type')
     Sv = echopype.calibrate.compute_Sv(ed, **range_kwargs)
-    mvbs = echopype.preprocess.compute_MVBS(Sv)
-    # plot = echopype.visualize.create_echogram(mvbs)
-    # assert isinstance(plot, FacetGrid) is True
+    mvbs = echopype.preprocess.compute_MVBS(Sv, ping_time_bin='1S')
```

Comment:
How about if we default to 10s here? ping_time interval at 1S is a common setting so we're not reducing much with ping_time_bin='1S'

Reply:
I put it as '1S' because the data I'm currently using is very small, and I end up having only 1 point, and the plotting doesn't work.